src/App.tsx

```diff
@@ -1,10 +1,53 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { SortEnum } from './types/sort';
 
-const USER_ID = 0;
+const USER_ID = 10589;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [select, setSelect] = useState('all');
+  const [error, setError] = useState(false);
```

Comment:
```suggestion
  const [isError, setIsError] = useState(false);
```
Fix all cases if present
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig